app/components/vendor_outage_alert_component.rb

```diff
@@ -0,0 +1,42 @@
+class VendorOutageAlertComponent < BaseComponent
+  include LinkHelper
+
+  def initialize(vendors:, only_if_all: false, context: 'default')
+    @vendors = vendors
+    @only_if_all = only_if_all
+    @context = context
```

Comment:
should we validate `context` here? or just let the missing translations catch it if context is an unknown value

Reply:
> should we validate `context` here? or just let the missing translations catch it if context is an unknown value

Yeah, my intention was to gracefully fall back to `default` if `context` doesn't exist, rather than having to duplicate between this file and the locale file. However, with `i18n-tasks` comments below, we sort of have to anyways.

I still think I'd rather keep it the way it is, though it's a good reminder for me to add a test case for this behavior.

Reply:
> it's a good reminder for me to add a test case for this behavior.

Added in 2b225bf.